aten/src/ATen/xpu/XPUEvent.h

```diff
@@ -128,7 +125,7 @@ struct TORCH_XPU_API XPUEvent {
     }
   }
 
-  float elapsed_time(const XPUEvent& other) const {
+  double elapsed_time(const XPUEvent& other) const {
```

Comment:
why double? align with other device type?

Reply:
Align to frontend API. refer to https://github.com/pytorch/pytorch/blob/26e5572dd24acd69b08eb94577f8796a480c9fe0/torch/csrc/xpu/Event.cpp#L100